### PR TITLE
Add draft for 14.4 release notes

### PR DESCRIPTION
**Pull request by oliverguenther**
[#56198] 14.4 Release notes prepared

https://community.openproject.org/work_packages/56198

### CODE REVIEW

**Comment by corinnaguenther** [comment]
Hi @ba1ash, could you please review the section about the extended API authentication?
And @mereghost, could you please review the section about the improved error messages?
Thanks a lot! 🙏

**Review by ba1ash — APPROVED** [review]
Looks good to me @corinnaguenther.

**Comment by mereghost** [comment]
@corinnaguenther I'd avoid using `AMPF` in the text and go with the long version `Automatically Managed Project Folders`, besides that looks perfect! :D Thanks for taking care of that.

**Comment by corinnaguenther** [comment]
Hi @MayaBerd, the release notes are ready to be reviewed in total – the two technical sections have already been reviewed by our dev colleagues. If you have any screenshots or docs-links you think would help understanding the features, I would be happy to include them. Thanks a lot in advance for your time and effort and have a wonderful vacation!

File: docs/release-notes/14-4-0/README.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+## Important technical updates
+
+### Extend API authentication to accept JWT issued by OpenID provider to other client
+
+OpenProject 14.4 introduces a new feature that allows OpenID clients, such as Nextcloud servers, to use access tokens obtained from an OpenID provider, like Keycloak, as an authentication mechanism for the OpenProject API. This enhancement **enables users to skip the OAuth grant flow**, streamlining the authentication process.
+
+With this feature, the OpenProject API will validate access tokens issued by the OpenID provider (Keycloak) by checking the token's signature and authenticating the user using the sub claim value. This integration ensures secure and efficient API authentication for OpenID clients.
+
+For more details, see https://community.openproject.org/wp/55643.
```
Comment (MayaBerd) [inline docs/release-notes/14-4-0/README.md]: Maybe also link to the documentation here: /docs/api/introduction/#oidc-provider-generated-jwt-as-a-bearer-token

File: docs/release-notes/14-4-0/README.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+Additionally, the logging structure has been enhanced to provide more detailed, tagged entries for better traceability.
+
+For more details, see https://community.openproject.org/wp/56861.
+
+## Important feature changes
+
+### Personal settings: Dark mode
+
+Dark mode for OpenProject is finally here! In the 'My account' section under 'Settings', there is an **option labeled 'Mode' where users can now select 'Dark (Beta).'** – as an alternative to the light mode. When the dark mode is selected, the change applies only to that user, not to the entire instance.
```
Comment (MayaBerd) [inline docs/release-notes/14-4-0/README.md]:
```suggestion
Dark mode for OpenProject is finally here! In the '[My account](../../getting-started/my-account/#profile-settings)' section under 'Settings', there is an **option labeled 'Mode' where users can now select 'Dark (Beta).'** – as an alternative to the light mode. When the dark mode is selected, the change applies only to that user, not to the entire instance. 
```

File: docs/release-notes/14-4-0/README.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+![News setting for dark mode in OpenProject, displayed in dark mode](openproject-14-4-dark-mode.png)
+
+Please note that this feature has been tested extensively, and we believe it works well. However, users might encounter issues that we haven't seen. Therefore, it will remain in beta for now, until we resolve all potential errors and ensure there are no issues.
+
+For more details, see https://community.openproject.org/wp/36233.
+
+### Project attributes: Separate permissions for viewing and editing
+
+In OpenProject 14.4, new permissions for managing project attributes provide project admins with more control over who can view and edit these attributes.
```
Comment (MayaBerd) [inline docs/release-notes/14-4-0/README.md]:
```suggestion
In OpenProject 14.4, new permissions for managing [project attributes](../../user-guide/projects/project-settings/project-attributes/) provide project admins with more control over who can view and edit these attributes.
```

File: docs/release-notes/14-4-0/README.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+To keep the activity view cleaner when multiple changes are made to long text custom fields, OpenProject 14.4 now **displays updates to long custom fields in a similar format to description updates**. As a user watching the activity, you will see an indication of changes to long text custom fields, making it easier to manage and review activities.
+
+In the activity tab, updates to long text custom fields will be displayed as in the following example – 'Remarks' being a custom field:
+
+![Example screenshot showing how changes on a custom field will now look like in the work package activity tab](Custom_field__long_text__changed.png)
+
+Clicking on the "Details" link will take the user to the diff view, which is also used for work package descriptions and project status explanations.
+
+<!--more-->
```
Comment (MayaBerd) [inline docs/release-notes/14-4-0/README.md:127]: You could add the link to the docs: /user-guide/projects/project-settings/custom-fields/ and/or
/docs/system-admin-guide/custom-fields/

**Review by MayaBerd — COMMENTED** [review]
@corinnaguenther I've added a few suggestions linking to the docs. 

Please also keep an eye on this bug, which came into 14.4 last week. (https://community.openproject.org/projects/documentation/work_packages/55299), it should be added after it's merged.

**Pull request merged**